vnext/Microsoft.ReactNative/Views/TextInputViewManager.cpp

```diff
@@ -277,21 +280,7 @@ void TextInputShadowNode::registerEvents() {
     }
   });
 
-  m_controlKeyDownRevoker =
-      control.KeyDown(winrt::auto_revoke, [=](auto &&, xaml::Input::KeyRoutedEventArgs const &args) {
-        if (args.Key() == winrt::Windows::System::VirtualKey::Enter && !args.Handled()) {
-          folly::dynamic eventDataSubmitEditing = {};
-          if (m_isTextBox) {
-            eventDataSubmitEditing = folly::dynamic::object("target", tag)(
-                "text", react::uwp::HstringToDynamic(control.as<xaml::Controls::TextBox>().Text()));
-          } else {
-            eventDataSubmitEditing = folly::dynamic::object("target", tag)(
-                "text", react::uwp::HstringToDynamic(control.as<xaml::Controls::PasswordBox>().Password()));
-          }
-          GetViewManager()->GetReactContext().DispatchEvent(
-              tag, "topTextInputSubmitEditing", std::move(eventDataSubmitEditing));
-        }
-      });
+  registerPreviewKeyDown();
```

Comment:
Note this will conflict with @stecrain changes
Please bear with us while we navigate this mess of CI issues, it will take some time to flush the pending PRs

Reply:
That change is already in, btw. Also, not sure if it conflicts. 
FYI, Eric: https://github.com/microsoft/react-native-windows/pull/7545

Reply:
No conflicts. Just pushed the rebase.